### PR TITLE
New parent pom

### PR DESCRIPTION
**Pull request by gjoranv**
@aressem Please verify that the addition to `screwdriver/release-java-artifacts.sh` is sufficient to have this released. I could not find any other relevant mentions of `container-dependency-versions`, for example.

### CODE REVIEW

File: dependency-versions/pom.xml [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+
+        <!-- Dependencies used internally in Vespa, not visible for users -->
+        <airline.vespa.version>0.9</airline.vespa.version>
+        <antlr.vespa.version>3.5.3</antlr.vespa.version>
+        <antlr4.vespa.version>4.11.1</antlr4.vespa.version>
+        <apache.httpclient.vespa.version>4.5.14</apache.httpclient.vespa.version>
+        <apache.httpcore.vespa.version>4.4.16</apache.httpcore.vespa.version>
+        <apache.httpclient5.vespa.version>5.2.1</apache.httpclient5.vespa.version> <!-- WARNING: sync cloud-tenant-base-dependencies-enforcer/pom.xml -->
+        <apache.httpcore5.vespa.version>5.2.2</apache.httpcore5.vespa.version> <!-- WARNING: sync cloud-tenant-base-dependencies-enforcer/pom.xml -->
```
Comment (bjorncs) [inline dependency-versions/pom.xml:54]: Stale comment?

File: dependency-versions/pom.xml [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        <asm.vespa.version>9.3</asm.vespa.version>
+
+        <!-- Athenz dependencies. Make sure these dependencies match those in Vespa's internal repositories -->
+        <athenz.vespa.version>1.11.34</athenz.vespa.version> <!-- WARNING: sync cloud-tenant-base-dependencies-enforcer/pom.xml -->
```
Comment (bjorncs) [inline dependency-versions/pom.xml:58]: Stale comment?

File: dependency-versions/pom.xml [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+        <jjwt.vespa.version>0.11.5</jjwt.vespa.version>
+        <jna.vespa.version>5.11.0</jna.vespa.version>
+        <junit.vespa.version>5.8.1</junit.vespa.version>
+        <junit.platform.vespa.version>1.8.1</junit.platform.vespa.version>
+        <maven-archiver.vespa.version>3.6.0</maven-archiver.vespa.version>
+        <maven-wagon.vespa.version>2.10</maven-wagon.vespa.version>
+        <mockito.vespa.version>4.0.0</mockito.vespa.version>
+        <netty.vespa.version>4.1.94.Final</netty.vespa.version>
+        <netty-tcnative.vespa.version>2.0.61.Final</netty-tcnative.vespa.version>
+        <onnxruntime.vespa.version>1.13.1</onnxruntime.vespa.version> <!-- WARNING: sync cloud-tenant-base-dependencies-enforcer/pom.xml -->
```
Comment (bjorncs) [inline dependency-versions/pom.xml:89]: Stale comment?

File: dependency-versions/pom.xml [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+        <zero-allocation-hashing.vespa.version>0.16</zero-allocation-hashing.vespa.version>
+        <zookeeper.client.vespa.version>3.8.0</zookeeper.client.vespa.version>
+
+
+        <!-- Maven plugins -->
+        <maven-assembly-plugin.vespa.version>3.3.0</maven-assembly-plugin.vespa.version>
+        <maven-bundle-plugin.vespa.version>5.1.9</maven-bundle-plugin.vespa.version>
+        <maven-compiler-plugin.vespa.version>3.10.1</maven-compiler-plugin.vespa.version>
+        <maven-core.vespa.version>3.8.7</maven-core.vespa.version>
+        <maven-dependency-plugin.vespa.version>3.6.0</maven-dependency-plugin.vespa.version> <!-- NOTE: When upgrading, also update explicit versions in tenant base poms! -->
```
Comment (bjorncs) [inline dependency-versions/pom.xml:106]: Stale comment?

**Review by bjorncs — DISMISSED** [review]

File: screwdriver/release-java-artifacts.sh [hunk 5]
```diff
@@ -61,6 +61,7 @@ export FACTORY_VESPA_VERSION=$VESPA_RELEASE
 
 COMMON_MAVEN_OPTS="$VESPA_MAVEN_EXTRA_OPTS --no-snapshot-updates --settings $(pwd)/screwdriver/settings-publish.xml --activate-profiles ossrh-deploy-vespa -DskipTests"
 TMPFILE=$(mktemp)
+mvn $COMMON_MAVEN_OPTS  -pl :dependency-versions -DskipStagingRepositoryClose=true deploy 2>&1 | tee $TMPFILE
 mvn $COMMON_MAVEN_OPTS  -pl :container-dependency-versions -DskipStagingRepositoryClose=true deploy 2>&1 | tee $TMPFILE
```
Comment (aressem) [inline screwdriver/release-java-artifacts.sh:65]: This line:
`mvn $COMMON_MAVEN_OPTS  -pl :container-dependency-versions -DskipStagingRepositoryClose=true deploy 2>&1 | tee $TMPFILE`
must be moved down below L70 and altered to:
`mvn $COMMON_MAVEN_OPTS  -pl :container-dependency-versions -DskipStagingRepositoryClose=true -DstagingRepositoryId=$STG_REPO deploy`

If not we are creating multiple staging repos and the dependency-versions will be in a separate orphaned staging repo.

**Review by aressem — CHANGES_REQUESTED** [review]

File: screwdriver/release-java-artifacts.sh (hunk 5 above)
Comment (gjoranv) [reply screwdriver/release-java-artifacts.sh]: Thanks for spotting my copy/paste and providing a fix!

**Review by bjorncs — APPROVED** [review]

**Comment by gjoranv** [comment]
@aressem PTAL

**Review by aressem — APPROVED** [review]

**Pull request merged**